.github/workflows/ci.yml

```
@@ -58,6 +58,7 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-latest
     needs: build
+    timeout-minutes: 90
```

```suggestion
    timeout-minutes: 30
```

Last run was 9 min, maybe we could lower this to 30?